### PR TITLE
LPS-166729

### PR DESCRIPTION
**Pull request by alinedoleron**
Ticket: https://issues.liferay.com/browse/LPS-166729

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

File: modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/field-types/FieldTypeList.es.js [hunk 1]
```diff
@@ -96,6 +96,46 @@ const FieldTypeList = ({
...
+		else {
+			document.getElementById(
+				'screenReaderSearchResult'
+			).innerText = sub(Liferay.Language.get('search-field-is-empty'), [
+				filteredFieldTypes.length,
+				keywords,
+			]);
+		}
+	}
+
```
Comment (cgmonte) [inline modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/field-types/FieldTypeList.es.js:138]: Getting elements by ID is usually faster than other methods for querying the DOM, but we can still avoid getting the same element twice by "storing" the element we need in a variable before further usages. So, before line 99, I would add:

`const screenReaderSearchResult = document.getElementById('screenReaderSearchResult');`

And then replacing all the subsequent calls to `document.getElementById()` with references to the previously created variable.

**Review by cgmonte — CHANGES_REQUESTED** [review]
Hi, Aline! I just made a non-critical suggestion to alter the code. Please, let me know what you think a bout it :)

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

**Review by cgmonte — APPROVED** [review]

**Comment by alinedoleron** [comment]
Manually sent to https://github.com/brianchandotcom/liferay-portal/pull/127951

**Pull request closed without merging**